includes/admin/class-wc-admin-notices.php

```diff
@@ -79,7 +79,7 @@ public static function remove_all_notices() {
 	 * Reset notices for themes when switched or a new version of WC is installed.
 	 */
 	public static function reset_admin_notices() {
-		if ( ! current_theme_supports( 'woocommerce' ) && ! in_array( get_option( 'template' ), wc_get_core_supported_themes() ) ) {
+		if ( ! current_theme_supports( 'woocommerce' ) ) {
 			self::add_notice( 'theme_support' );
 		}
```

Comment:
I think we can probably remove this notice entirely if all themes should work with WooCommerce.

Reply:
Doesn't need to be in this PR. Just observing.